network/p2p/network.go

```diff
@@ -252,6 +253,8 @@ func (n *Network) genNetworkMessage(channel network.Channel, event interface{},
 		return nil, fmt.Errorf("could not encode event: %w", err)
 	}
 
+	p := binstat.EnterTimeVal("~3net:wire<3payload2message", "", int64(len(payload)))
+
```

Comment:
do we want to keep binstat always on for this method?

Reply:
@ramtinms There is a whole bunch of them in the code so far:

```
$ find . -type f | egrep "\.go" | egrep -v test | xargs egrep "binstat.Enter" | wc -l  
      44
$ find . -type f | egrep "\.go" | egrep -v test | xargs egrep "binstat.Enter"        
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:r:Backend.Has", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Has", "")
./module/mempool/stdmap/backend.go:     p0 := binstat.EnterTime("~7Backend.Add/ID", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:w:Backend.Add", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Add", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:w:Backend.Rem", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Rem", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:w:Backend.Adjust", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Adjust", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:r:Backend.ByID", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.ByID", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:w:Backend.Run", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Run", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:r:Backend.Size", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Size", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:r:Backend.All", "")
./module/mempool/stdmap/backend.go:     p := binstat.EnterTime("~4lock:w:Backend.Clear", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Clear", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:r:Backend.Hash", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.Hash", "")
./module/mempool/stdmap/backend.go:     p1 := binstat.EnterTime("~4lock:w:Backend.RegisterEjectionCallbacks", "")
./module/mempool/stdmap/backend.go:     p2 := binstat.EnterTime("~7Backend.RegisterEjectionCallbacks", "")
./module/mempool/stdmap/backend.go:     p := binstat.EnterTime("~7Backend.reduce", "")
./module/mempool/stdmap/queues.go:      p1 := binstat.EnterTime("~4lock:w:Backend.Run(Queues)", "")
./module/mempool/stdmap/queues.go:      p2 := binstat.EnterTime("~7Backend.Run(Queues)", "")
./module/mempool/stdmap/blockbycollections.go:  p1 := binstat.EnterTime("~4lock:w:Backend.Run(BlockByCollections)", "")
./module/mempool/stdmap/blockbycollections.go:  p2 := binstat.EnterTime("~7Backend.Run(BlockByCollections)", "")
./network/codec/cbor/encode.go: p := binstat.EnterTime(fmt.Sprintf("%s%s:%d", via, what, code), "")
./network/codec/cbor/decoder.go:        p := binstat.EnterTime("~3net:strm>1", "")
./network/codec/cbor/encoder.go:        p := binstat.EnterTimeVal("~3net:strm<2", "", int64(len(data)))
./network/codec/cbor/decode.go: p := binstat.EnterTimeVal(fmt.Sprintf("%s%s:%d", via, what, code), "", int64(len(data)))
./network/codec/cbor/codec.go:  p := binstat.EnterTime("~3net:wire<2(cbor)envelope2payload", "")
./network/codec/cbor/codec.go:  p := binstat.EnterTime("~3net:wire>3(cbor)payload2envelope", "")
./network/codec/json/encode.go: p := binstat.EnterTime(fmt.Sprintf("%s%s:%d", via, what, code), "")
./network/codec/json/decoder.go:        p := binstat.EnterTime("~3net:strm>1", "")
./network/codec/json/encoder.go:        p := binstat.EnterTimeVal("~3net:strm<2", "", int64(len(env.Data)))
./network/codec/json/decode.go: p := binstat.EnterTimeVal(fmt.Sprintf("%s%s:%d", via, what, env.Code), "", int64(len(env.Data)))
./network/codec/json/codec.go:  p := binstat.EnterTime("~3net:wire<2(json)envelope2payload", "")
./network/codec/json/codec.go:  p := binstat.EnterTime("~3net:wire>3(json)payload2envelope", "")
./network/p2p/network.go:       p := binstat.EnterTimeVal("~3net:wire<3payload2message", "", int64(len(payload)))
./network/p2p/readSubscription.go:              p := binstat.EnterTimeVal("~3net:wire>1protobuf2message", "", int64(len(rawMsg.Data)))
./network/p2p/middleware.go:    p := binstat.EnterTime("~3net:wire<4message2protobuf", "")
./model/flow/identifier.go:     p1 := binstat.EnterTime("~6MakeID/Fingerprint", "")
./model/flow/identifier.go:     p2 := binstat.EnterTimeVal("~6MakeID/ComputeHash", "", int64(len(data)))
```

Reply:
So far I have made the assumption that it's fine to sprinkle them in as long as they are disabled by default (please see other comment regarding what 'disabled' means). But that assumption could be wrong... I guess this is the right place to discuss.